packages/smart-contracts/src/contracts/Erc20ConversionProxy.sol

```diff
@@ -19,14 +19,24 @@ contract Erc20ConversionProxy {
     chainlinkConversionPath = ChainlinkConversionPath(_chainlinkConversionPathAddress);
   }
 
-  // Event to declare a transfer with a reference
+  // Event to declare a conversion with a reference
   event TransferWithConversionAndReference(
     uint256 amount,
     address currency,
     bytes indexed paymentReference,
     uint256 feeAmount,
     uint256 maxRateTimespan
   );
+  
+  // Event to declare a transfer with a reference
```

Comment:
won't modifying this potentially cause issues with the existing contract?

Reply:
Sorry I forgot to commit my comments on the review. This event declaration missing explains why on mainnet we don't see the interpretation of the first event sent by ERC20ConversionProxy. Cf. my comment below.